css/homepage.css

```diff
@@ -0,0 +1,84 @@
+@import url(/css/navbar.css);
+
+
+
+
+/* slider css */
+.slider-box {
+    width : 80%;
+    height: 100%;
+    max-height: 100vh;
+    position  : relative;
+
+    margin    : 2rem auto;
+    overflow: hidden;
+}
+
+#slider {
+    width     : 100%;
+    display   : inline-flex;
+    transition: 0.7s;
+}
+
+#slider img {
+    /* overflow: hidden; */
+    width: 100%;
+}
+
+.indicators {
+    /* position : absolute; */
+    left     : 50%;
+    bottom   : 1.3rem;
+    transform: translateX(-50%);
+}
+
+.indicators span {
+    display      : inline-block;
+    width        : 15px;
+    height       : 15px;
+    border-radius: 50%;
+    background   : rgba(255, 255, 255, 0.5);
+    cursor       : pointer;
+}
+
+.active {
+    background: #ffff !important;
+}
+/* slider css end */

```

Comment:
no need to specify end of a certain class styling. The beginning of the new class marks the end of the previous one.